src/interactive/Modal.jsx

```diff
@@ -130,6 +130,7 @@ export class ModalComponent extends React.Component {
 	render() {
 		const {
 			className,
+			modalClassName,
```

Comment:
Were you able to try passing in a class to `className` and then target the modal? It should be supported by creating a class at the top level.

```.css
.customClassFromConsumer .view--modal {
  ...
}
```

and then passing the `.customClassFromConsumer` to the `className` prop. ex:
```jsx
<Modal className='customClassFromConsumer' {...props} />
```

or if you're using css modules as 

```jsx
import styleModule from './styles'

<Modal className={styleModule.customClassFromConsumer} {...props} />
```

Reply:
I tried that but it doesn't work, because the `view--modal` class also gets hashed

Reply:
The css winds up looking like this, so it wont target view--modal
<img width="534" alt="Screen Shot 2019-03-21 at 12 13 08 PM" src="https://user-images.githubusercontent.com/1933722/54767033-c0b7d800-4bd2-11e9-86d4-1c4e9fab988c.png">

Reply:
I see, it looks like we're hashing the nested classes too. In that case it should work by indicating that view module is global. ex:
```scss
.customClassFromConsumer {
  :global {
    .view--modal {
      ...
    }
  }
}
```

Reply:
It's better explained in the css loader scope docs here -> https://github.com/webpack-contrib/css-loader#scope